### PR TITLE
Add spelling correction for export

### PR DESCRIPTION
**Pull request by adrien-berchet**
https://github.com/search?q=exprot&type=code => 13589 hits
https://github.com/search?q=exprots&type=code => 3018 hits
https://github.com/search?q=exproted&type=code => 4277 hits

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -15065,6 +15065,9 @@ expriences->experiences
 exprimental->experimental
 expropiated->expropriated
 expropiation->expropriation
+exprot->export
+exproted->exported
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:15069]:
```suggestion
exproted->exported
exproting->exporting
```

Comment (adrien-berchet) [reply codespell_lib/data/dictionary.txt:15069]: Good catch, thanks

**Review by vikivivi — APPROVED** [review]
LGTM

**Comment by luzpaz** [comment]
TIA!

**Pull request merged**